drivers/sensor/ms5607/Kconfig

```diff
@@ -0,0 +1,60 @@
+# MS5607 pressure sensor configuration options
+
+#
+# Copyright (c) 2019 Thomas Schmid <tom@lfence.de>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
```

Comment:
Could shorten to

```
# Copyright (c) 2019 Thomas Schmid <tom@lfence.de>
# SPDX-License-Identifier: Apache-2.0
```

See https://github.com/zephyrproject-rtos/zephyr/pull/20280.